Formula/mirrord.rb

```diff
@@ -0,0 +1,20 @@
+class Mirrord < Formula
+  desc "Connect your local process and your cloud environment"
+  homepage "https://mirrord.dev"
+  url "https://github.com/metalbear-co/mirrord/releases/download/2.13.0/mirrord_mac_universal.zip"
```

Comment:
Don't we have a 'latest' path?

Reply:
i think we need the url to have the version number